examples/approveAllowances.ts

```diff
@@ -0,0 +1,104 @@
+import { BigNumber, constants, ethers } from "ethers";
+import { config as dotenvConfig } from "dotenv";
+import { resolve } from "path";
+import { Chain } from "../src";
+import { usdcAbi } from "./abi/usdcAbi";
+import { ctfAbi } from "./abi/ctfAbi";
+
+dotenvConfig({ path: resolve(__dirname, "../.env") });
+
+export const MUMBAI_CONTRACTS = {
+    Exchange: "0x4bFb41d5B3570DeFd03C39a9A4D8dE6Bd8B8982E",
+    Collateral: "0x2E8DCfE708D44ae2e406a1c02DFE2Fa13012f961",
+    Conditional: "0x7D8610E9567d2a6C9FBf66a5A13E9Ba8bb120d43",
+};
+
+export const MAINNET_CONTRACTS = {
+    Exchange: "0x4bFb41d5B3570DeFd03C39a9A4D8dE6Bd8B8982E",
+    Collateral: "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
+    Conditional: "0x4D97DCd97eC945f40cF65F87097ACe5EA0476045",
+};
```

Comment:
nit: instead of having the addresses here we can call this utils function: https://github.com/Polymarket/clob-order-utils/blob/main/src/networks.ts#LL22C20-L22C20